public/index.php

```diff
@@ -63,10 +63,16 @@
      */
     include APP_PATH . '/config/loader.php';
 
+    /**
+     * Include Middleware
+     */
+    include APP_PATH . '/config/middleware.php';
```

Comment:
the implementation of the middleware does not seems right. please stop to use bad examples from stack overlow.
you should never need to include files. Phalcon uses composer and PSR4 standard, and it should not required includes.
Once again.... stack overflow sucks... next copy/past = block stackoverlow